<?xml version="1.0" encoding="UTF-8"?>
<launch>
  
    <node name="object_detect_haar"	pkg="tf_ros_detection"      type="object_detect_haar"  	respawn="true" 	output="screen"><!--    launch-prefix="xterm -e gdb !args"-->
                <remap from="cam_in" 			to="/stereo_cam_node/image_raw"/>
                <remap from="cam_left_info" 	      	to="/stereo_cam_node/left/camera_info"/>
                <remap from="cam_right_info" 		to="/stereo_cam_node/right/camera_info"/>
		<param name="root_dir"	type="string" 	value="/home/interface/softwares/tensorflow_models/"   />
		<param name="labels_dir"	type="string" 	value="/mscoco_label_map2.pbtxt" /> 
		<param name="models_dir"	type="string" 	value="/home/ubuntu/catkin_ws/src/tf_ros_detection/nodes/haarcascade_frontalface_default.xml" />
 		<param name="threshold_Score" value="0.7" />
 		<param name="threshold_IOU" value="0.8" />
     </node>

</launch>
